ldap-connector/processes/query.mod

```diff
@@ -0,0 +1,78 @@
+[Ivy]
+17D5C3ADA54E7B5A 9.3.1 #module
+>Proto >Proto Collection #zClass
+qy0 query Big #zClass
+qy0 B #cInfo
+qy0 #process
+qy0 @AnnotationInP-0n ai ai #zField
+qy0 @TextInP .type .type #zField
+qy0 @TextInP .processKind .processKind #zField
+qy0 @TextInP .xml .xml #zField
+qy0 @TextInP .responsibility .responsibility #zField
+qy0 @StartSub f0 '' #zField
+qy0 @EndSub f1 '' #zField
+qy0 @GridStep f3 '' #zField
+qy0 @PushWFArc f2 '' #zField
+qy0 @PushWFArc f4 '' #zField
+qy0 @PushWFArc f6 '' #zField
+qy0 @CallSub f5 '' #zField
+>Proto qy0 qy0 query #zField
+qy0 f0 inParamDecl '<com.axonivy.connector.ldap.LdapQuery ldapQuery> param;' #txt
+qy0 f0 inParamInfo 'ldapQuery.description=query used to retrieve data from AD' #txt

```

Comment:
I would write
query used to retrieve data from LDAP server

because your ldap-connector is really generic. you only know about LDAP. the Active Directory and other tools understand the LDAP protocol. The Active Directory is only one of them.

So basically. You should never use the term Active Directory in your connector.
But in your tests its fine. Because you test against an active directory.
And also in demo its fine!